### PR TITLE
🌱added tests for exec.go

### LINKED ISSUE #3211: Increase the test coverage with unit tests
### What do you want to happen?

By looking at the coveralls reports we can see that we have some implementations which could be better covered. We do not cover 100% of the code but would be great increase the coverage and ensure that we have no red and orange files/dirs, see: https://coveralls.io/github/kubernetes-sigs/kubebuilder?branch=master

Example: 

We could add more tests for cmd_helpers.go, completiton.go and version.go:

Screenshot 2023-02-08 at 14 17 13

### Extra Labels

_No response_

### PR DESCRIPTION
**Pull request by khareyash05**
added unit tests for `pkg/plugin/exec.go`

Fixes #3211

### CODE REVIEW

**Review by camilamacedo86 — COMMENTED** [review]
/ok-to-test

You MUST assign to the CLA otherwise we cannot get this one merged.
Also, see that this change does not affected end-users therefore, the right emoji is `🌱`. Could you please change the title?

File: pkg/plugin/util/exec_test.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package util
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestRunCmd(t *testing.T) {
```
Comment (camilamacedo86) [inline pkg/plugin/util/exec_test.go:10]: See that we use the following libs to do the tests. Could you please follow the same standard to do this one?

> 	. "github.com/onsi/ginkgo/v2"
> 	. "github.com/onsi/gomega"

Example:
https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugin/util/helpers_test.go

Comment (khareyash05) [reply pkg/plugin/util/exec_test.go:10]: Sure will do that

**Comment by khareyash05** [comment]
@camilamacedo86 Added imports to the file

**Comment by khareyash05** [comment]
@camilamacedo86 Could you please help me with the failing test. I am new to this repo and getting no idea from the build info

**Comment by camilamacedo86** [comment]
Hi @khareyash05 ,

>@camilamacedo86 Added imports to the file

Thank you for the contribution. See that it is not only about add the imports but indeed use the libs and do the tests in the same format, approach that we have been doing in the project. Could you please check the other tests implemented and do it at the same away?

**Pull request closed without merging**